positions/USDC-Supervisor-Product-Management.md

```diff
@@ -0,0 +1,347 @@
+# These dates are also used to determine whether a position is upcoming, open,
+# or closed. Here's how we decide:
+#
+# The job is upcoming if:
+#   opens is empty OR
+#   opens is in the future
+#
+# The job is open if:
+#   opens is in the past OR
+#   closes is in the future
+#
+# The job is closed if:
+#   closes is in the past
+#
+# If either opens or closes is empty, missing, or not a date, the position is
+# considered to be upcoming.
+
+# If the position can also close when the maximum number of applications are
+# received, include the maximum number of applications here. Remove this line
+# or set the value to 0 if the job does not have an application cap.
+#🔻🔻🔻🔻🔻
+max applications: 0
+
+# Is this a permanent or term appointment? Use either "perm" or "term". This
+# will be used to fill in the appointment type on the page with consistent
+# language.
+#🔻🔻🔻🔻🔻
+appointment type: term
+
+# Put the GS grade this position is being advertised at. For SES positions, set
+# the level to 20.
+#🔻🔻🔻🔻🔻
+gs: 14
+# The information you put here will be used to automatically pull salary range
+# information.
+#
+# ⚠️⚠️⚠️ IMPORTANT NOTE: The automatic salary ranges are based on 100% remote
+# positions and use the lowest and highest locality areas. If this position is
+# not 100% remote, you will need to delete the automatic salary range and input
+# it yourself down below. Look for 💰💰💰 to indicate where to do that.
+#
+# The salary range data is stored in this file:
+#
+#   _data/pay_ranges.yml
+#
+# The data file will need to be updated each year to reflect any pay changes.
+
+# List key objectives here. Key objectives and sub-bullets will be displayed in
+# the order they are listed here. You do not need to include key objective
+# numbers. They will be added automatically.
+#
+# Each key objective starts on a new line and must start with a dash. It does
+# not need to include a number at the beginning. A key objective can be spread
+# across multiple lines if you want.
+#
+# To add sub-bullets to your key objective, add a new line below the key
+# objective and indent it with spaces. (The number of spaces does not matter, as
+# long as it is more than one). Then, add a dash and your sub-bullet.
+# Sub-bullets can also take up multiple lines as long as all of the lines are
+# indented the same amount so they line up.
+#
+# The placeholder information below shows an example of how to format the key
+# objectives. This example indents sub-bullets with two spaces:
+#🔻🔻🔻🔻🔻
+key objectives:
+  - Provide product management coaching, mentorship, and professional development opportunities; support employee wellbeing for Digital Corps Fellows
+    - Steward the Fellow’s personal professional development by identifying and providing appropriate training, developmental assignments, and/or support focusing on their specific discipline of product management.
+    - Utilize previous experience as a product management practitioner and supervisor to: 
```

Comment:
The ":" on this line is creating an error on the markdown. The same apply tp the whole bullet list. There are more : symbols.